src/communication/xcWebSocketSubscriber.js

```diff
@@ -42,20 +42,29 @@ define(["rx"], function (Rx) {
 
 
```

Comment:
it's strange to have a public method called send (getEventToSend) in a subscriber method !